pkg/stream/xprotocol/connpool_multiplex.go

```diff
@@ -177,6 +178,8 @@ func (p *poolMultiplex) NewStream(ctx context.Context, receiver types.StreamRece
 		return host, nil, types.Overflow
 	}
 
+	mosnctx.WithValue(ctx, types.ContextUpstreamConnectionID, strconv.FormatUint(activeClient.codecClient.ConnID(), 10))
```

Comment:
add this logic in other connpools

Reply:
Help to check.
types.ContextUpstreamConnectionID
types.VarUpstreamConnectID